### ISSUE TITLE
difference between refined anchors and proposals

### ISSUE THREAD

**Issue by dingjiansw101** [sfzhang15/RefineDet#78]
Hi, I think your work is very inspiring. But I have a question.
What is the difference between refined anchors and proposals from the RPN?

**Comment by sfzhang15** [comment]
@dingjiansw101 
Thanks. Both of them provide better regression initialization. However, Refined anchors just filter out easy negatives (>0.01), while proposals from RPN do NMS and choose top N anchors. We want the ARM to have RPN-like function. Besides, you may be confused by the difference bwtween one-stage and two-stage, we think the the key aspect to distinguish the one-stage and two-stage detector is whether it has the region-wise operation (e.g., ROIPooling).

**Comment by dingjiansw101** [comment]
Thank you for your answer. So, the main difference is the post-processing after getting refined anchors or proposals?
And at the same time, whether you can understand this way, in the two-stage detector, each ROI corresponds to a feature region. In one stage detector, there are several anchors for each feature region. And in a two-stage detector, the position of the feature region is also calculated by the ROI.

**Comment by sfzhang15** [comment]
@dingjiansw101 
Yes, the region-wise opreation (e.g., ROIPooling) will align the feature according to RoI.
